DOCS/new-license-workflow.md

```diff
@@ -17,23 +24,154 @@ Ask the submitter for any additional info needed, preferably via the Github issu
+    * The title of the license should be wrapped in `<titleText></titleText>` tags. The value here should match the value in the `name` attribute in the `license` tag above.
+    * Every paragraph should be wrapped in `<p></p>` tags, including the text in the `<titleText>` tag. Text in list tags (see below) do not need to be wrapped in `<p>` tags.
+    * Lists—either ordered (numbered/lettered) or unordered (bullets)—must be formatted with `list`, `item`, and `bullet` tags. Lists can be nested. It's much easier to show this than explain it, so here's a basic example (see also [CC-BY-NC-ND-3.0-IGO](../src/CC-BY-NC-ND-3.0-IGO.xml) for a real world example):
+```
+<list>
+    <item>
+        <bullet>1.</bullet> text 1
+    </item>
+    <item>
+        <bullet>2.</bullet> text 2
+        <list>
+            <item>
+                <bullet>*</bullet> text *
+            </item>
+            <item>
+                <bullet>-</bullet> text -
+            </item>
+        </list>
+    </item>
+    <item>
+        <bullet>3.</bullet> text 3
+        <list>
+            <item>
+                <bullet>a.</bullet> text a
+            </item>
+            <item>
+                <bullet>b.</bullet> text b
+                <list>
+                    <item>
+                        <bullet>i.</bullet> text i
+                    </item>
+                    <item>
+                        <bullet>ii.</bullet> text ii
+                    </item>
+                </list>
+            </item>
+        </list>
+    </item>
+</list>
+```
+5. Save the file and [commit it](https://docs.github.com/en/github/committing-changes-to-your-project) to your clone of the repository.
+
+You're done! Yes, there may be more advanced cases where more work than this is required. If you don't add licenses often, it's unlikely the core team will have assigned you something like that, but if they did or you have any questions at all, please ask! They love to help!
+
+### Test locally
+
+When you send the pull request (below), our [CI/CD](https://en.wikipedia.org/wiki/CI/CD) tool [Travis](https://travis-ci.org/github/spdx/license-list-XML) will run all of the tests to make sure everything is OK.
+
+However, that can take quite a bit of time since it runs the tests for every single license in the list. If you make a mistake, you may be waiting a while to learn about it. Often it's much faster to test things locally first. There are three ways to do this:
+
+#### Use your browser to test the XML file
+
+Most web browsers "speak" XML, which means they're a quick and easy way to tell whether you have valid XML in your file. If you've messed something up (forgotten a closing tag, for instance), the browser will give you an error. If everything is A-OK, the browser will show you your XML file.
+
+To test this, either point your browser at the XML file in your file system using [`file://`](https://en.wikipedia.org/wiki/File_URI_scheme) or drag/drop the XML file onto your browser window.
```

Comment:
I'll add a note here just clarifying that this option tests the validity of the XML file, but doesn't test whether it matches the .txt file. I assume people would know this, but might be worth flagging it so they know that full testing will include the subsequent step(s) too.